lib/teiserver/player/session_registry.ex

```diff
@@ -35,4 +35,8 @@ defmodule Teiserver.Player.SessionRegistry do
       start: {__MODULE__, :start_link, []}
     )
   end
+
+  def get_count() do
```

Comment:
I’d go with `count` to mimic the `Registry` api.

Reply:
Fixed